.github/workflows/pip-requirements.yml

```diff
@@ -68,10 +67,42 @@ jobs:
           message_success: |
             The changes to the 'requirements-fixed.txt' have been added to this PR.
             All further changes to any of the requirements file will automatically be applied as long an :$REACT_EMOTE: is present.
-          script_call: scripts/compile-requirements.sh scripts/requirements-fixed.txt
           git_diff_root: ncs/nrf
           diff_path: scripts
           diff_file: requirements-fixed.txt
           reaction_emote: "${{ env.REACT_EMOTE }}"
-          git_user_name: "NordicBuilder"
-          git_user_email: "NordicBuilder@github.com"
+          git_user_name: "Nordic Builder"
+          git_user_email: "ci-help@nordicsemi.no"
```

Comment:
This needs to be the e-mail that is registered with the Nordic Builder account, otherwise github will complain / the push will not show up correctly in the PR.
So it needs to be 'pylon@nordicsemi.no'

Reply:
change the email in nordicBuilder profile?

Reply:
IMO if we change email I don't think we should start doing it here. It should be separate PR to main first (then we can backport in case needed). There are probably more repos, actions and branches to change it. Personally I would prefer handling emails somehow on infra level (no / minimal code changes). For example perhaps it is possible to configure multiple emails and/or setup forwarding.